### PR TITLE
Add content view

### PR DESCRIPTION
**Pull request by itkg-nanne**
https://github.com/open-orchestra/open-orchestra-cms-bundle/pull/2081
https://github.com/open-orchestra/open-orchestra-model-bundle/pull/727
https://github.com/open-orchestra/open-orchestra-model-interface/pull/284
https://github.com/open-orchestra/open-orchestra-user-bundle/pull/140
https://github.com/open-orchestra/open-orchestra-mongo-libs/pull/68
https://github.com/open-orchestra/open-orchestra/pull/1136
https://github.com/open-orchestra/open-orchestra-provision/pull/42
https://github.com/open-orchestra/open-orchestra-provision-docker/pull/3
https://github.com/open-orchestra/open-orchestra-docs/pull/242

### CODE REVIEW

File: ApiBundle/Controller/ContentController.php [hunk 1]
```diff
@@ -124,15 +103,20 @@ public function duplicateAction(Request $request)
...
-        $newContentId = null;
-        foreach (array_keys($frontLanguages) as $language) {
-            $content = $this->findOneContent($contentId, $language);
-            if ($content instanceof ContentInterface) {
-                $duplicateContent = $this->get('open_orchestra_backoffice.manager.content')->duplicateContent($content, $newContentId);
-                $newContentId = $duplicateContent->getContentId();
-                $this->dispatchEvent(ContentEvents::CONTENT_DUPLICATE, new ContentEvent($duplicateContent));
+        $contentType = $this->get('open_orchestra_model.repository.content_type')->findOneByContentTypeIdInLastVersion($content->getContentType());
+
+        if (!is_null($contentType) && $contentType->isDefiningVersionable()) {
```
Comment (alavieille) [inline ApiBundle/Controller/ContentController.php]: why only versionable content type ?

File: BackofficeBundle/Resources/public/ecmascript/OpenOrchestra/Application/View/Content/ContentFormView.js [hunk 2]
```diff
@@ -0,0 +1,88 @@
...
+ * @class ContentFormView
+ */
+class ContentFormView extends mix(AbstractFormView).with(FormViewButtonsMixin) 
+{
+    /**
+     * Initialize
+     * @param {Form}   form
+     * @param {Array}  name
+     */
+    initialize({form, name, contentTypeId, language, siteLanguageUrl, contentId = null}) {
```
Comment (alavieille) [inline BackofficeBundle/Resources/public/ecmascript/OpenOrchestra/Application/View/Content/ContentFormView.js]: js doc

File: BackofficeBundle/Resources/public/ecmascript/OpenOrchestra/Application/View/Content/ContentFormView.js [hunk 3]
```diff
@@ -0,0 +1,88 @@
...
+        });
+        this.$el.html(template);
+        this._$formRegion = $('.form-edit', this.$el);
+        super.render();
+
+        return this;
+    }
+
+    /**
+     * Redirect to edit user view
```
Comment (alavieille) [inline BackofficeBundle/Resources/public/ecmascript/OpenOrchestra/Application/View/Content/ContentFormView.js]: user ?

**Comment by itkg-nanne** [comment]
update done @alavieille

File: WorkflowAdminBundle/EventSubscriber/AddContentWorkflowStatusSubscriber.php [hunk 4]
```diff
@@ -0,0 +1,40 @@
...
+
+namespace OpenOrchestra\WorkflowAdminBundle\EventSubscriber;
+
+use Symfony\Component\EventDispatcher\EventSubscriberInterface;
+use OpenOrchestra\Backoffice\Event\ContentFormEvent;
+use OpenOrchestra\Backoffice\ContentFormEvents;
+
+/**
+ * Class AddContentWorkflowStatusSubscriber
+ *
```
Comment (alavieille) [inline WorkflowAdminBundle/EventSubscriber/AddContentWorkflowStatusSubscriber.php]: remove this line

**Comment by alavieille** [comment]
Add cron in documentation and provision docker and provision vagrant

**Pull request merged**